SAFU.md

```diff
@@ -0,0 +1,78 @@
+# Simple Arrangement for Funding Upload
+
+## Overview
+
+This Simple Arrangement for Funding Upload (the “[SAFU](https://jumpcrypto.com/safu-creating-a-standard-for-whitehats/)” or “Arrangement”) is intended as a simple yet extensible way to specify a post-exploit policy for whitehats, particularly rewards and distributions. It is based in the SAFU Framework designed by [Jump Crypto](https://jumpcrypto.com/).
+
+This Arrangement attempts to address the following issues in during active vulnerabilities and post-exploits:
+
+- **Legal uncertainty**: No clear grace period during which the hacker can declare themselves as a white hat. No formal guarantees as the team can decide at any time to take legal action or not.
+- **Lack of clarity**: What to do with the funds secured from an exploit? Where should the white hat transfer the secured funds? Is there a compensation/reward promised for securing affected users’ funds?
+- **Execution risk**: Conflicting proposals and stressful negotiation may arise during and after the exploit of a vulnerability, leading to additional confusion and uncertainty between the parties involved.
+
+## Concepts
+
+- **Dropbox Address (”Dropbox”)**: An address or contract to which funds taken from the protocol should be deposited. In the case of contracts, a Dropbox can be *automatic*, handling claims and rewards on a per-depositor basis without human input, or *conditional*, requiring additional input such as governance approvals or identity verification (KYC).
+- **Deposit Interval:** the grace period in which a sender must deposit funds in the Dropbox after removing them from the protocol.
+- **Claim Delay:** a minimum waiting period before a sender may claim rewards. We recommend at least 24 hours, during which the extent of an exploit will become clear.
+- **Sender Claim Interval:** a maximum waiting period after which the protocol may reclaim the sender’s reward, to avoid leaving funds stranded in the contract.
+- B**ounty Percent**: Pro-rata share of funds secured that are claimable by the whitehats.
+- **Bounty Cap**: Maximum amount of tokens that a whitehat can claim after securing the vulnerable funds.
+
+## Statement for Whitehats
+
+Tharsis Labs Ltd. (the ”Team”) commits to not pursue legal action against white hats who act in accordance with the Arrangement for vulnerabilities found in the Evmos blockchain (chain ID: 9001).
```

Comment:
Could you please add the link to the Arrangement?

Reply:
If it is referring to this doc please add "this" since it is cited with a different name.